projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -310,6 +314,22 @@ export class AppExtensionService implements RuleContext {
     return { presets };
   }
 
+  loadBulkActionsDropdown(config: ExtensionConfig): any {
+    const elements = this.loader.getElements<any>(config, 'features.bulk-actions-dropdown');
+    if (!elements.length) {
+      return null;
+    }
+
+    let presets = {};
+    presets = this.filterDisabled(mergeObjects(presets, ...elements));
+
+    const metadata = this.appConfig.config['content-metadata'] || {};
```

Comment:
What this particular part of the code does? Looks like straight copy paste from another method

Reply:
removed